CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased][unreleased]
+
+### Added
+
+- This Changelog
+
+### Changed
+
+- `bem-mod-*` attributes now treats value as expression.
```

Comment:
maybe "`bem-mod-*` attributes value now treated as expression." ?
attributes not independent. it processed in `bemDirective`